sourmash_lib/__main__.py

```diff
@@ -938,6 +939,47 @@ def do_search():
         sig.save_signatures([streamsig], args.output)
 
 
+def convert(args):
+    "Convert YAML to JSON"
+    
+    import sourmash_lib.signature
+
+    parser = argparse.ArgumentParser("""
+Ensure that signature files in YAML (old format) are converted to signature files in JSON (new format).
+The JSON-for-sure signature files are created in the same directory as the YAML-may-be files and are added
+the extension ".json".
+""")
+    parser.add_argument('--minified', action="store_true",
+                        help='Store the JSON minified (uses less space but is less human-readable) or not.')
+    parser.add_argument('-f', '--force',
+                        action="store_true",
+                        help="Overwrite existing files.")
+    parser.add_argument('filename', nargs='*', help='Name of YAML file with signatures.')
+    
+    args = parser.parse_args(args)
+
+    if args.minified:
+        kwargs = {'indent': None,
+                  'sort_keys': False}
+    else:
+        kwargs = {}
+
+    for fn in args.filename:
```

Comment:
Since it is a command line, one could delegate this to the shell:
```
sourmash convert $(find mydirectory/ --name '*.sig')
```
(although it will generally not work for large collections of signature files)